packages/@okta/vuepress-site/docs/guides/build-provisioning-integration/prepare-api/index.md

```diff
@@ -110,6 +110,17 @@ A best practice is to use a generated globally unique identifier (GUID) for this
 
 Okta user management requires that your SCIM API supports an `active` attribute for each user resource that can be set to `true` or `false` to denote a resource as "active" or "inactive".
 
+## Example implementations
+
+There are numerous examples of how to implement SCIM servers and applications. Here are a few to get you started:
+
+- [scimify](https://toolkit.okta.com/apps/scimify/) - A PHP application for Okta that supports both SCIM 1.1 and SCIM 2.0 servers with operations for /Users, /Groups and /ServiceProviderConfig endpoints
```

Comment:
replace dashes with &mdash;

also comma after /Groups - we use the serial comma

Reply:
Bad copy and pasting! Bad!